src/swarm-tui/Swarm/TUI/Controller.hs

```diff
@@ -1303,13 +1317,18 @@ validateREPLForm s =
     CmdPrompt _
       | otherwise ->
           let ctxs = Contexts (topCtx ^. defTypes) (topCtx ^. defReqs) (topCtx ^. tydefVals)
-              result = processTerm' ctxs uinput
-              theType = case result of
-                Right (Just pt) -> Just (pt ^. processedSyntax . sType)
-                _ -> Nothing
+              (theType, errSrcLoc) = case readTerm' defaultParserConfig uinput of
+                Left err ->
+                  let ((x1, _y1), (x2, _y2), _msg) = showErrorPos err
```

Comment:
Aren't these (line, column)?  i.e. should we be ignoring `x1` and `x2` and using `y1` and `y2` instead?

Reply:
It seems so. I copied the code that was using this function and assumed the convention is that `y` is lines.